typekits/rtt_tf2_msgs/package.xml

```diff
@@ -0,0 +1,52 @@
+<package>
+  <name>rtt_tf2_msgs</name>
+  <version>2.9.1</version>
+  <description>
+
+    Provides an rtt typekit for ROS nav_msgs messages.
+
+    It allows you to use ROS messages transparently in
+    RTT components and applications.
+
+    This package was automatically generated by the
+    create_rtt_msgs generator and should not be manually
+    modified.
+
+    See the http://ros.org/wiki/nav_msgs documentation
+    for the documentation of the ROS messages in this
+    typekit.
+
+  </description>
+  <maintainer email="orocos-dev@lists.mech.kuleuven.be">Orocos Developers</maintainer>
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>tf2_msgs</build_depend>
+  <run_depend>tf2_msgs</run_depend>
+
+  <run_depend>rtt_tf</run_depend>
+  <build_depend>rtt_tf</build_depend>
```

Comment:
The typekit package cannot depend on `rtt_tf`. `rtt_tf` should depend on `rtt_tf2_msgs`.